server/app/controllers/SupportController.java

```diff
@@ -9,20 +9,30 @@
 import play.mvc.Http;
 import play.mvc.Result;
 import views.support.UnconfirmedIdcsEmailBugView;
+import views.support.UnsupportedBrowserView;
 
 public class SupportController extends Controller {
```

Comment:
Not blocking: Consider preemptively marking this as final to make the choice to allow subclassing explicit rather than implicit.